subsys/bluetooth/host/adv.c

```diff
@@ -1592,10 +1592,43 @@ void bt_le_adv_resume(void)
 int bt_le_ext_adv_get_info(const struct bt_le_ext_adv *adv,
 			   struct bt_le_ext_adv_info *info)
 {
+	if (adv == NULL) {
+		LOG_DBG("adv is NULL");
+		return -EINVAL;
+	}
+
```

Comment:
Add a check that `adv` is from the Host allocated `adv_pool`? Otherwise, later check do not help guard from application declared memory.

Reply:
Did consider that myself; added. 

Should we omit the NULL check then?

Reply:
NULL check is redundant as `IS_ARRAY_ELEMENT` also checks for NULL.